hbase/src/main/resources/hbase/bin/hirb.rb

```diff
@@ -0,0 +1,189 @@
+#
```

Comment:
this seems to be a file from standard HBase distribution.
is there a reason why this can use the latest one: https://github.com/apache/hbase/blob/master/bin/hirb.rb
and is there a reason why we need to include a copy here and not use something on hbase.home?